Drivers/NER/src/cascadiamc.c

```diff
@@ -0,0 +1,407 @@
+/**
+ * @file cascadiamc.c
+ * @author Hamza Iqbal
+ * @brief Source file for Motor Controller Driver
+ * @version 0.1
+ * @date 2023-08-22
+ * 
+ * @copyright Copyright (c) 2023
+ * 
+ */
+
+#include "cascadiamc.h"
+
+void cascadiamc_init(cascadiamc_t* mc)
```

Comment:
Is anything supposed to happen in this function? Do we need to initialize the CAN bus if not initialized, link a specific CAN bus, etc?

Reply:
I kinda threw that in there and forgot about it thats my bad

Reply:
I do like the init function tho, might be good to throw in the CAN line that we are talking to